lib/cylc/scheduler.py

```diff
@@ -714,7 +740,7 @@ def configure_suite(self, reconfigure=False):
             else:
                 # Remove database created by previous runs
                 if os.path.isdir(pri_db_path):
-                    shutil.rmtree(pri_db_path)
+                    rmtree(pri_db_path)
```

Comment:
Good spotting ... but why is/was this test necessary (how can `pri_db_path` be a dir?)